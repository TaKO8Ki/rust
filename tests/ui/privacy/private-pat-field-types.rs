mod foo {
    struct Bar;
    pub enum Foo {
        #[allow(private_interfaces)]
        A(Bar),
    }
}
fn foo_bar(v: foo::Foo) {
    match v {
        foo::Foo::A(_) => {} //~ ERROR type `Bar` is private
    }
}

fn main() {}
