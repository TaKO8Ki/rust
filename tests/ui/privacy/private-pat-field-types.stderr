error: type `Bar` is private
  --> $DIR/private-pat-field-types.rs:10:21
   |
LL |         foo::Foo::A(_) => {}
   |                     ^ private type
   |
help: consider using `..` to ignore the field with private type
   |
LL -         foo::Foo::A(_) => {}
LL +         foo::Foo::A(..) => {}
   |

error: aborting due to 1 previous error

